pom.xml

```
@@ -119,13 +119,77 @@
                                 <version>8.29</version>
                             </dependency>
                         </dependencies>
-                        <configuration>
+                        <configuration>     
                             <configLocation>strongbox-checks.xml</configLocation>
                             <consoleOutput>true</consoleOutput>
                         </configuration>
                     </plugin>
                 </plugins>
             </build>
         </profile>
+        <profile>
+            <id>Enforce-Java-11</id>
```

Please, change this to:
```suggestion
            <id>enforce-jdk11</id>
```

---

resolved